### PR TITLE
Update readme with Docker Swarm Variant information

### PR DESCRIPTION
**Pull request by trevorgetty**
Updating the readme documentation with the docker swarm orchestrator variants.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,29 @@ The [Autoscaler](https://github.com/Autoscaler/autoscaler) service provides on-d
...
 Manifest of the components which make up the AutoScaler:
 
-* autoscale-container
+Note: As the Autoscaler has two orchestrator variants currently, the manifest is split into 
+- Marathan or Docker Swarm Assets
+- Common Assets.
+
+
+### Marthan Variants
+* autoscale-marathan-container
```
Comment (andyreidz) [inline README.md]:
```suggestion
* autoscale-marathon-container
```

**Comment by andyreidz** [comment]
@trevorgetty Thanks for the PR, can you complete a [CLA](https://github.com/CAFapi/CAFapi.github.io/blob/master/EntIT_CCLA%202018.docx) and email it to me?

File: README.md [hunk 2]
```diff
@@ -4,14 +4,29 @@ The [Autoscaler](https://github.com/Autoscaler/autoscaler) service provides on-d
...
 
 Manifest of the components which make up the AutoScaler:
 
-* autoscale-container
+Note: As the Autoscaler has two orchestrator variants currently, the manifest is split into 
+- Marathan or Docker Swarm Assets
+- Common Assets.
+
+
+### Marthan Variants
```
Comment (dermot-hardy) [inline README.md:16]: Marthan --> Mar**a**th**o**n

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (trevorgetty) [reply README.md:16]: I will go through all those spelling mistakes, that's what I get for copy/paste!

**Comment by trevorgetty** [comment]
Hi guys, the CLA linked, has a section in it: 
_If you have not already done so, please complete and sign, then scan and email a pdf file of this Agreement to **[insert e-mail address here]**._

You may wish to insert your email address I think, I will update my local one with Andy's email?  
And I am getting our legal to agree to this, thanks. trev.

**Comment by thilagasanthosh** [comment]
@dermot-hardy ,

This PR is with you . Do we need to include this change to release?
Please confirm.

**Comment by dermot-hardy** [comment]
Check with @andyreidz what the situation with this is but if there is still a legal issue then no it shouldn't  hold up the release.

**Comment by andyreidz** [comment]
Pleas go ahead with the release without this PR.

**Comment by xbreizh** [comment]
Closing it as inactive and no information about a DCO signature

**Pull request closed without merging**